pulsar-flink-connector/src/test/java/org/apache/flink/streaming/connectors/pulsar/internal/SourceSinkUtilsTest.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.flink.streaming.connectors.pulsar.internal;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+/** Unit test for SourceSinkUtils. */
+public class SourceSinkUtilsTest {
+    @Test
+    public void testRangeTopicBelongTo() {
+        String topic = "persistent://cme_dev/market_data_mbo_v1/345_0-partition-0";
```

Comment:
Wondering do we have a test case that cover the issue ?

Reply:
This test is added for the key hash range issue.